API.md

```diff
@@ -6,35 +6,31 @@
 
 - `id: PeerId`, the id of the local instance.
 - `libp2p: Libp2p`, instance of the local network stack.
-- `datastore: Datastore`, instance of the local database (`IpfsRepo.datastore`)
+- `blockstore: Datastore`, instance of the local database (`IpfsRepo.blockstore`)
 
 Create a new instance.
 
-### `getBlock(key, cb)`
 
-- `key: Multihash`
-- `cb: Function`
-
-Fetch a single block.
+### `getStream(key)`
 
-> Note: This is safe guarded so that the network is not asked
-> for blocks that are in the local `datastore`.
+- `key: Multihash`
 
-### `getBlocks(keys, cb)`
+Returns a source `pull-stream`. Values emitted are the received blocks.
```

Comment:
Is this going to return several blocks for one key? Seems that there is a need for get(key), get(keys), getStream(keys)